cmd/kill.go

```diff
@@ -64,3 +85,50 @@ func killJob(cmd *cobra.Command, args []string) {
 		}
 	}
 }
+
+func killTasks(cmd *cobra.Command, args []string) {
+	log.Infof("Killing task [Env:%s Role:%s Name:%s Instance:%s]\n", *env, *role, *name, *instance)
+
+	//Set jobKey for the tasks to be killed.
+	task := realis.NewTask().
+		Environment(*env).
+		Role(*role).
+		Name(*name)
+
+	//Check that instance number is passed
+	if instance == nil {
+		log.Fatalln("Instance number not found. Please pass a valid instance number. If you want to pass multiple instances, please pass them as space separated integer values")
+	}
+
+	/*
+	* In the following block, we convert instance numbers, which were passed as strings, to integer values
+	* After converting them to integers, we add them to a slice of type int32.
+	 */
+	var intErr error
+	var instanceNumber int
```

Comment:
We don't need line 107 & 108.

Reply:
We need the two variables for the loop on line 113. I tried without them, golang won't allow

Reply:
in line 127, you can do 

instanceNumber, intErr := strconv.Atoi(splitString[i]) right?

Reply:
Nope. Apparently golang requires variables to be used in a loop to be defined before the loop. I tried to do it the way you want, but it throws compilation errors